From e877545e05210d1cc9b802d4f34aa24e197986fb Mon Sep 17 00:00:00 2001
From: Alona Solntseva <alona.solntseva@tandemg.com>
Date: Wed, 09 Feb 2022 17:48:18 +0200
Subject: [PATCH] [SW-67906] erom: change image location to sram

Move erom to sram memory to allow proper images work

Change-Id: I7b453e9e039112bb2fbd081507e0030f2abd0562
Signed-off-by: Alona Solntseva <alona.solntseva@tandemg.com>
---

diff --git a/app/ext_rom/common.h b/app/ext_rom/common.h
index b895c04..dbbbf51 100644
--- a/app/ext_rom/common.h
+++ b/app/ext_rom/common.h
@@ -6,7 +6,7 @@
 #ifndef COMMON_EXT_ROM_H_
 #define COMMON_EXT_ROM_H_
 
-#define EXT_ROM_START		(0)
+#define EXT_ROM_START		(0x27FE2000)
 #define EXT_ROM_TEXT_LENGTH	(70*1024)
 /* No header bytes. Hence the text base is same as erom start address */
 #define EXT_ROM_TEXT_BASE	(EXT_ROM_START)
